pkgs/by-name/lu/luastatus/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+# Native Build Inputs
+, cmake
+, pkg-config
+, makeWrapper
+# Dependencies
```

Comment:
Instead of `xorg`, list all your dependencies explicitly: `libxcb libX11 xcbutil xcbutilwm libXdmcp libXau` (of course one per line)

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/256033/commits/a303b8979d8f94e8bbe55741dfc716312b687328